lib/services/build.ts

```diff
@@ -422,6 +422,10 @@ export class BuildService implements Project.IBuildService {
 
 			this.$loginManager.ensureLoggedIn().wait();
 
+			if(this.$project.projectData.WPSdk && this.$project.projectData.WPSdk === "8.0" && helpers.versionCompare(this.$project.projectData.FrameworkVersion,"3.7.0") >= 0) {
+				this.$logger.warn("Your project is using Cordova %s, so you can use WPSdk 8.1. You can change your WPSdk version with '$ appbuilder prop set WPSdk 8.1'", this.$project.projectData.FrameworkVersion);
```

Comment:
Your project targets Apache Cordova %s which lets you use the Windows Phone 8.1 SDK when building your apps. You can change your target Windows Phone SDK by running $ appbuilder prop set WPSdk 8.1